spec.html

```diff
@@ -13615,7 +13615,7 @@ <h1>Applying the `**` Operator</h1>
         <li>If _base_ &lt; 0 and _base_ is finite and _exponent_ is finite and _exponent_ is not an integer, the result is *NaN*.</li>
       </ul>
       <emu-note>
-        <p>The result of _base_ `**` _exponent_ when _base_ is *1* or *-1* and _exponent_ is *+Infinity* or *-Infinity* differs from IEEE 754-2008. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2008 specified *1*. The historical ECMAScript behaviour is preserved for compatibility reasons.</p>
+        <p>The result of _base_ `**` _exponent_ when _base_ is *1* or *-1* and _exponent_ is *+Infinity*, *-Infinity* or *NaN* differs from IEEE 754-2008. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2008 specified *1*. The historical ECMAScript behaviour is preserved for compatibility reasons.</p>
```

Comment:
i'm confused; how did the first edition of ECMAScript specify a result for the exponentiation operation, which was added in the 7th edition?

Reply:
`Math.exp`?

Reply:
Ah, duh, i confused myself.

Reply:
Up until 28bdf079, that sentence was about `Math.pow(x, y)`, which did exist in the first edition.